tests/unit/stat_test.ts

```diff
@@ -25,12 +25,8 @@ Deno.test(
 
     const tempFile = Deno.makeTempFileSync();
     const tempInfo = Deno.statSync(tempFile);
-    let now = Date.now();
     assert(tempInfo.atime !== null && now - tempInfo.atime.valueOf() < 10000);
     assert(tempInfo.mtime !== null && now - tempInfo.mtime.valueOf() < 10000);
```

Comment:
This is still using `now`? Maybe we should just increase these from 10_000 to 60_000